.github/workflows/sdk-build-container.yml

```diff
@@ -12,25 +12,24 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-name: SDK - Build Base Container
+name: SDK - App Builder
 concurrency:
-  group: sdk-build-container-${{ github.ref }}
+  group: sdk-app-builder-${{ github.ref }}
   cancel-in-progress: true
 
 on:
   workflow_dispatch:
   push:
     branches:
-      - "main"
-  release:
-    types: [published, edited]
+      - main
+  pull_request:
+    branches:
+      - main
 
 jobs:
   build-container:
+    name: Building image
     runs-on: ubuntu-latest
-    #strategy:
-    #  matrix:
-    #    platform: [amd64, aarch64]
```

Comment:
should we still keep this? we will hopefully find a fix soon...

Reply:
Will hopefully be a multiarch build in case it is fixed, not a strategy.